Tasks/PackerBuildV0/DefaultTemplates/custom.managed.linux.template.json

```diff
@@ -0,0 +1,68 @@
+{
+  "variables": {
+    "client_id": "{{env `client_id`}}",
+    "client_secret": "{{env `client_secret`}}",
+    "subscription_id": "{{env `subscription_id`}}",
+    "tenant_id": "{{env `tenant_id`}}",
+    "object_id": "{{env `object_id`}}",
+    "resource_group": "{{env `resource_group`}}",
+    "storage_account": "{{env `storage_account`}}",
+    "image_url": "{{env `image_url`}}",
```

Comment:
Does image_url work here? What I mean is if you provide a VHD url, will packer be able to generate a managed image? Or will it only create a VHD?

Reply:
As mentioned in my comment below, I have tested with my own custom managed and unmanaged templates which have dependencies on other files and folders in the repo. 
Can validate/test source managed image as a part of a separate PR.